samples/dotnet/Models.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.Extensions.Logging;

```

Comment:
Remove the unnecessary `using`

All you should need:
using Microsoft.Extensions.Logging;
using StackExchange.Redis;
using System.Collections.Generic;